geoai/universat.py

```
@@ -0,0 +1,209 @@
+    "s1-asc",
+    "s1flair",
+    "s2",
+    "s2_4band",
+    "s2l1c",
+    "s2flair",
+    "s2withaerosol",
+    "s3",
+    "l8",
+    "l8forestnet",
+    "l7",
+    "al" "os",
```

Ruff (0.15.21)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)